src/test/java/example/Example.java

```diff
@@ -58,7 +63,10 @@ public WebSecurityConfiguration getWebSecurityConfiguration() {
         }
     }
 
-    public static final class ExampleApplication extends Application<ExampleConfiguration> {
+    /**
+     * Example Web Application.
+     */
+    public static class ExampleWebApplication extends Application<ExampleConfiguration> {
 
         private final WebSecurityConfiguration webSecurityDefaults = WebSecurityConfiguration.builder()
```

Comment:
yes, but it wouldn't be a clear example of how to use this in the common case, which is a web application. i admit it's starting to lose that narrow focus as we add more ETE tests through them, but i use this specific block to generate the README examples